pyxem/utils/azimuthal_utils.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2024 The pyXem developers
+#
+# This file is part of pyXem.
+#
+# pyXem is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyXem is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pyXem.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import numpy as np
+
+from shapely import Polygon, box
+import shapely
+import numba
+
+
+@numba.njit
+def slice_radial_integrate(img, factors, factors_slice, slices, npt_rad, npt_azim):
```

Comment:
could this function be private? and if so, could this module by private?

Reply:
Yes it should be!  If we are going to be better about this we should probably start here :)